testsuite/integration/src/test/java/org/keycloak/testsuite/AssertEvents.java

```diff
@@ -133,6 +137,17 @@ public ExpectedEvent expectLogin() {
                 .session(isUUID());
     }
 
+    public ExpectedEvent expectLogin(String username) {
```

Comment:
Just call "expectLogin().detail(Details.USERNAME, usrname)" to override the expected username, no need to add an additional method